### PR TITLE
OKD 4.5, 4.6: update build procedure

### PR DESCRIPTION
**Pull request by vrutkovs**
Use repos from image, install CRI-O from Fedora modules and install open-vm-tools.

For some reason these are out of sync when 4.5 and 4.6 were created

### CODE REVIEW

File: ci-operator/jobs/openshift/release/openshift-release-release-4.5-periodics.yaml [hunk 1]
```diff
@@ -539,11 +539,14 @@ periodics:
...
                   curl -L "${tar_url}" | tar xf - -C /srv/repo/ && \\
-                  rm -rf /etc/yum.repos.d/* && \\
+                  rm -rf /etc/yum.repos.d && \\
+                  ostree --repo=/srv/repo checkout "${ostree_ref}" --subpath /usr/etc/yum.repos.d --user-mode /etc/yum.repos.d && \\
+                  sed -i 's/enabled=0/enabled=1/g' /etc/yum.repos.d/fedora-updates-testing-modular.repo && \\
                   rpm-ostree db list --repo /srv/repo "${ostree_ref}" > /tmp/packages && \\
-                  curl http://base-${BRANCH//\./-}-rhel8.ocp.svc > /etc/yum.repos.d/rhel8.repo && \\
-                  if ! grep -q cri-o /tmp/packages; then yumdownloader -y --disablerepo=* --enablerepo=rhel-8* --destdir=/tmp/rpms cri-o cri-tools; fi && \\
-                  if ! grep -q glusterfs-fuse /tmp/packages; then yumdownloader -y --disablerepo=* --enablerepo=rhel-8* --destdir=/tmp/rpms attr glusterfs glusterfs-client-xlators glusterfs-fuse glusterfs-libs psmisc; fi && \\
+                  dnf module enable -y cri-o:1.17 && \\
```
Comment (smarterclayton) [inline ci-operator/jobs/openshift/release/openshift-release-release-4.5-periodics.yaml]: Fedora will have 1.18 soon, right?  We need 1.18 once we do rebase (generally).

Comment (vrutkovs) [reply ci-operator/jobs/openshift/release/openshift-release-release-4.5-periodics.yaml]: Correct, I'll switch to 1.18 once Lokesh prepares a new module (I'll ping him around rebase time to prepare the module)

**Pull request merged**